packages/terra-date-picker/tests/jest/__snapshots__/DatePicker.test.jsx.snap

```diff
@@ -1,639 +1,11 @@
-exports[`test should render a date picker with disabled dates 1`] = `
```

Comment:
Due to the shallow rendering, this snapshot of jest isn't rendered correctly. It just contains `<datePicker />`. We may want to use deep render to render all children inside it.